_unittest/test_42_configuration_files.py

```diff
@@ -0,0 +1,134 @@
+# standard imports
+import os
+
+from _unittest.conftest import BasisTest
+from pyaedt import Hfss
+from pyaedt import Icepak
+from pyaedt import Q2d
+from pyaedt import Q3d
+
+
+# Import required modules
+# Setup paths for module imports
+
+
+test_project_name = "coax_setup_solved"
+test_field_name = "Potter_Horn"
+sbr_file = "poc_scat_small"
+q3d_file = "via_gsg"
+ipk_name = "Icepak_test"
+
+
+class TestClass(BasisTest, object):
+    def setup_class(self):
+        # set a scratch directory and the environment / test data
+        BasisTest.my_setup(self)
+        self.aedtapp = BasisTest.add_app(self, project_name=test_project_name)
+        # self.field_test = BasisTest.add_app(self, project_name=test_field_name)
+        # self.sbr_test = BasisTest.add_app(self, project_name=sbr_file)
+        self.q3dtest = BasisTest.add_app(self, project_name=q3d_file, application=Q3d)
+        self.q2dtest = Q2d(projectname=q3d_file)
+        self.icepak = BasisTest.add_app(self, project_name=ipk_name, application=Icepak)
+
+    def teardown_class(self):
+        BasisTest.my_teardown(self)
+
+    def test_01_hfss_export(self):
+        self.aedtapp.mesh.assign_length_mesh("inner")
+        conf_file = self.aedtapp.configurations.export_config()
+        assert os.path.exists(conf_file)
+        import shutil
+
+        shutil.copy(conf_file, "C:\\temp\\test.json")
```

Comment:
@maxcapodi78 Instead of the `"C:\\temp\\test.json"`, you could use a python TemporaryDirectory from the `tempfile` package. I am not sure that this file is deleted at the end of the test.

Reply:
It was a forgotten debug line. Thanks for noticing it.